subsys/spm/Kconfig

```diff
@@ -12,33 +12,8 @@ config SPM
 	imply ARM_FIRMWARE_USES_SECURE_ENTRY_FUNCS
 
 if SPM
-
-choice
-	prompt "SPM build strategy"
-	default SPM_BUILD_STRATEGY_FROM_SOURCE
-
-config SPM_BUILD_STRATEGY_USE_HEX_FILE
-	# Mandatory option when being built through 'zephyr_add_executable'
-	bool "Use hex file instead of building SPM"
-
-if SPM_BUILD_STRATEGY_USE_HEX_FILE
-
-config SPM_HEX_FILE
-	# Mandatory option when being built through 'zephyr_add_executable'
-	string "SPM hex file"
-
-endif # SPM_USE_HEX_FILE
-
-config SPM_BUILD_STRATEGY_SKIP_BUILD
-	# Mandatory option when being built through 'zephyr_add_executable'
-	bool "Skip building SPM"
-
-config SPM_BUILD_STRATEGY_FROM_SOURCE
-	# Mandatory option when being built through 'zephyr_add_executable'
-	bool "Build from source"
-
-endchoice
-
+module=SPM
+source "${ZEPHYR_BASE}/../nrf/subsys/partition_manager/Kconfig.template.build_strategy"
```

Comment:
Why not just use `rsource ../partition_manager/Kconfig.template.build_strategy`?

Reply:
If i do that I get the following error:
```
ninja: error: build.ninja:14535: multiple rules generate /home/haam/ncs/nrf/subsys/partiti
on_manager/Kconfig.template.build_strategy [-w dupbuild=err]
```

Reply:
Ping @mbolivar

Reply:
Works on my machine @hakonfam 

```
diff --git a/subsys/spm/Kconfig b/subsys/spm/Kconfig
index 6ada6770..ae203994 100644
--- a/subsys/spm/Kconfig
+++ b/subsys/spm/Kconfig
@@ -13,7 +13,7 @@ config SPM
 
 if SPM
 module=SPM
-source "${ZEPHYR_BASE}/../nrf/subsys/partition_manager/Kconfig.template.build_strategy"
+rsource "../partition_manager/Kconfig.template.build_strategy"
 endif
 

```

Reply:
will try again

Reply:
https://github.com/zephyrproject-rtos/zephyr/pull/18123

This was the reason for the failure.

I promise to re-visit this when that PR is in :)

Reply:
> I promise to re-visit this when that PR is in :)
Deal